deploy/chart/templates/plugin.yaml

```diff
@@ -1,17 +1,20 @@
-{{- if .Values.plugin.enabled -}}
+{{- range .Values.nodepools -}}
+{{- if $.Values.plugin.enabled }}
+{{- $copy := deepCopy .csi }}
+{{- $csi := mergeOverwrite $.Values.csi $copy }}
 kind: DaemonSet
 apiVersion: apps/v1
 metadata:
-  name: csi-plugin
-  namespace: {{ .Release.Namespace }}
+  name: {{ .name }}
+  namespace: {{ $.Release.Namespace }}
 spec:
   selector:
     matchLabels:
-      app: csi-plugin
+      app: {{ .name }}
```

Comment:
I suggest use two labels
```
app: csi-plugin
nodepool: {{ $.key }}
```
so that we can select all the csi-plugin pods with kubectl while debugging.

Reply:
It makes sense.